common/src/main/java/com/wynntils/wc/custom/item/UnidentifiedItemStack.java

```diff
@@ -68,30 +70,49 @@ public UnidentifiedItemStack(ItemStack stack) {
         if (rarityMap == null) return;
 
         List<String> items = rarityMap.get(tier);
-        if (items == null) return;
+        if (items == null || items.isEmpty()) return;
+
+        tooltip.add(new TranslatableComponent("feature.wynntils.itemGuess.possibilities"));
+
+        Map<Integer, List<MutableComponent>> levelToItems = new TreeMap<>();
 
-        MutableComponent guesses = new TextComponent("");
         for (String item : items) {
+            ItemProfile profile = WebManager.getItemsMap().get(item);
```

Comment:
Based on WebManagers design, I assume ItemGuessFeature loads the items? (Just to confirm)

Reply:
It doesn't as of now, so I'll try to remedy that.